NoClassDefFoundError with sspi authentication

When "sspi" is enabled on a postgres server to connect using windows ad/single sign on, ili2pg raises an exception.
If sspi is diabled on the same server and "md5" is used (user/password based authentication), everything works fine.

```
java -Djava.security.auth.login.config=jaas.conf -jar ili2pg-4.4.2/ili2pg-4.4.2.jar --createEnumTxtCol --schemaimport --importTid --sqlEnableNull --createEnumTabs --createFk  --defaultSrsCode 2056  --noSmartMapping --dbusr user --dbhost host  --dbport 5432  --dbdatabase test --modeldir ili --dbschema vsa_dss_2015_d --log 00schema_VSA_DSS_2015_d.log VSA_DSS_2015_2_d_20170602.ili

Info: dburl <jdbc:postgresql://host:5432/test>
Info: dbusr <user>

Exception in thread "main" java.lang.NoClassDefFoundError: com/sun/jna/platform/win32/Win32Exception
        at java.lang.Class.forName0(Native Method)
        at java.lang.Class.forName(Unknown Source)
        at org.postgresql.core.v3.ConnectionFactoryImpl.createSSPI(ConnectionFactoryImpl.java:73)
        at org.postgresql.core.v3.ConnectionFactoryImpl.doAuthentication(ConnectionFactoryImpl.java:566)
        at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:222)
        at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)
        at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:194)
        at org.postgresql.Driver.makeConnection(Driver.java:450)
        at org.postgresql.Driver.connect(Driver.java:252)
        at java.sql.DriverManager.getConnection(Unknown Source)
        at java.sql.DriverManager.getConnection(Unknown Source)
        at ch.ehi.ili2db.base.AbstractJdbcMapping.connect(AbstractJdbcMapping.java:31)
        at ch.ehi.ili2db.base.Ili2db.connect(Ili2db.java:1795)
        at ch.ehi.ili2db.base.Ili2db.readSettingsFromDb(Ili2db.java:187)
        at ch.ehi.ili2db.AbstractMain.domain(AbstractMain.java:553)
        at ch.ehi.ili2pg.PgMain.main(PgMain.java:70)
Caused by: java.lang.ClassNotFoundException: com.sun.jna.platform.win32.Win32Exception
        at java.net.URLClassLoader.findClass(Unknown Source)
        at java.lang.ClassLoader.loadClass(Unknown Source)
        at sun.misc.Launcher$AppClassLoader.loadClass(Unknown Source)
        at java.lang.ClassLoader.loadClass(Unknown Source)
        ... 16 more
```

Note: skipping jaas.conf in the parameters triggers the same error

The server configuration seems to be fine, plain `psql` can be used to send queries to the server successfully.

Is there a java module missing locally?

--- a/build.gradle
+++ b/build.gradle
@@ -330,6 +330,8 @@ dependencies {
 	testImplementation group: 'org.hamcrest', name: 'hamcrest-library', version: '1.3'
 	ili2pgImplementation group: 'org.postgresql', name: 'postgresql', version: '42.1.4.jre6'
 	compileOnly group: 'org.postgresql', name: 'postgresql', version: '42.1.4.jre6' // add as compileOnly, so that eclipse sees it
+	ili2pgImplementation group: 'com.github.waffle', name: 'waffle-jna', version: '1.9.1'
+	compileOnly group: 'com.github.waffle', name: 'waffle-jna', version: '1.9.1' // add as compileOnly, so that eclipse sees it
 	ili2gpkgImplementation group: 'org.xerial', name: 'sqlite-jdbc', version: '3.8.11.2'
 	compileOnly group: 'org.xerial', name: 'sqlite-jdbc', version: '3.8.11.2' // add as compileOnly, so that eclipse sees it
 	ili2mysqlImplementation group: 'mysql', name: 'mysql-connector-java', version: '5.1.47'

